fdbserver/Knobs.cpp

```diff
@@ -459,6 +459,9 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( REPLACE_INTERFACE_CHECK_DELAY,                         5.0 );
 	init( COORDINATOR_REGISTER_INTERVAL,                         5.0 );
 	init( CLIENT_REGISTER_INTERVAL,                            600.0 );
+	init( CC_PEER_HEALTH_CHECKING_INTERVAL,                    60.0 );
```

Comment:
FYI: you can add if( randomize && BUGGIFY ) to make it happen more frequently in simulation.  
At least, test if changing knob can break your code.